admin_guide/cluster_metrics.adoc

```diff
@@ -16,298 +16,382 @@ As an OpenShift administrator, you may want to view the metrics from all
 containers and components in one user interface. The
 link:../architecture/infrastructure_components/kubernetes_infrastructure.html#kubelet[kubelet]
 exposes metrics that can be collected and stored in various back ends by
-link:https://github.com/GoogleCloudPlatform/heapster[Heapster]. This guide uses
-link:https://influxdb.com/[InfluxDB] for storage and
-link:http://grafana.org[Grafana] for visualization.
+link:https://github.com/GoogleCloudPlatform/heapster[Heapster]. This guide uses 
+link:https://github.com/hawkular/hawkular-metrics[Hawkular Metrics] as a metrics engine which 
+persists the data into a link:http://cassandra.apache.org/[Cassandra] database.
 
-[IMPORTANT]
-====
-This solution is a work in progress. As packaging improvements are made, these
-instructions will be simplified.
-====
+Once this is configured, metrics can then be viewable from from within the OpenShift console.
 
-== Node Configuration
+The current implementation has been more heavily focused on CPU and memory based metrics. More
+indepth metrics are expected in a future release.
```

Comment:
Not sure the docs are the right place to talk about future plans. Maybe just say CPU and memory metrics in the previous sentence?

"Once configured, CPU and memory-based metrics can be viewed from the web console."